src/data/complex/is_R_or_C.lean

```diff
@@ -526,6 +535,14 @@ lemma is_cau_seq_abs {f : ℕ → K} (hf : is_cau_seq abs f) :
 λ ε ε0, let ⟨i, hi⟩ := hf ε ε0 in
 ⟨i, λ j hj, lt_of_le_of_lt (abs_abs_sub_le_abs_sub _ _) (hi j hj)⟩
 
+@[simp, norm_cast] lemma of_real_prod {α : Type*} (s : finset α) (f : α → ℝ) :
+  ((∏ i in s, f i : ℝ) : K) = ∏ i in s, (f i : K) :=
+ring_hom.map_prod (@coe_hom K _) _ _
+
+@[simp, norm_cast] lemma of_real_sum {α : Type*} (s : finset α) (f : α → ℝ) :
+  ((∑ i in s, f i : ℝ) : K) = ∑ i in s, (f i : K) :=
+ring_hom.map_sum (@coe_hom K _) _ _
```

Comment:
Could you add lemmas for `ring_hom.map_finsupp_sum` here too?

Reply:
Also, I think this will work:
```suggestion
@[simp, norm_cast, to_additive] lemma of_real_prod {α : Type*} (s : finset α) (f : α → ℝ) :
  ((∏ i in s, f i : ℝ) : K) = ∏ i in s, (f i : K) :=
ring_hom.map_prod (@coe_hom K _) _ _
```

Reply:
It doesn't work: it complains about `comm_monoid ℝ` not being `add_comm_monoid ℝ`.

Reply:
Done!